app/src/main/java/com/example/cartoon/MainActivity.kt

```diff
@@ -59,19 +60,19 @@ class MainActivity : ComponentActivity() {
 
             // Check if no image is selected
             if (drawable == null) {
-                FancyToast.makeText(
-                    this@MainActivity,
-                    "No image selected",FancyToast.LENGTH_LONG,
-                    FancyToast.WARNING,true).show()
+                Toast.makeText(this@MainActivity, "No image selected", Toast.LENGTH_LONG).show()
 
             }else{  notification.sendNotification(
```

Comment:
sendNotification() have to be called inside try method of the saveCartoonedImage() after the saving process is completed by Glide.

Reply:
ok